community/index.md

```diff
@@ -0,0 +1,12 @@
+---
+sidebar_position: 1
+---
+
+# About conda-forge
+
+- [Governance](./governance)
+- [Join the team](./join)
+- [Current sub-teams](./subteams)
+- [Index of CFEPs](./cfep)
+- [Get in touch](./get-in-touch)
+- [Contracting information](./contracting)
```

Comment:
Maybe we could use cards here:

```
import DocCardList from "@theme/DocCardList";

<DocCardList />
```

You'll need this file to use `.mdx` as an extension.

Reply:
I will take a look :)

Reply:
Will leave as is for now, getting a:

<img width="605" alt="Screenshot 2024-02-23 at 12 32 54 AM" src="https://github.com/conda-forge/conda-forge.github.io/assets/3627835/00298b13-ed14-473a-889b-bb15060d08e5">